src/containers/Layout/Footer.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+    PageHeader,
+    PageHeaderTools,
+    PageHeaderToolsItem
+  } from "@patternfly/react-core";
+  import React from "react";
+  import { useHistory } from 'react-router-dom';
+  import logo from "../../assets/images/logo-white.png";
+  import github from "../../assets/images/githubLogo.png";
+  import chris from "../../assets/images/ChRISLogo.png"
+  
+  const Footer: React.FC = () => {
+    const history = useHistory();
+  
+    const pageToolbar = (
+        <React.Fragment>
+            <PageHeaderTools>
+                <PageHeaderToolsItem>
+                    <p className="footerText">An open-source neural network for disease detection created by 
+                    <a href="https://www.darwinai.com/" target="_blank" rel="noopener noreferrer"> DarwinAI</a>, 
+                    <a href="https://www.redhat.com/en" target="_blank" rel="noopener noreferrer"> RedHat</a>, and 
+                    <a href="https://www.childrenshospital.org/" target="_blank" rel="noopener noreferrer"> Boston Children's Hospital</a></p>
```

Comment:
I would suggest leaving the space outside of `<a>`, otherwise, it would underline it as a hyperlink as well.

```suggestion
                    <p className="footerText">An open-source neural network for disease detection created by&nbsp;
                    <a href="https://www.darwinai.com/" target="_blank" rel="noopener noreferrer">DarwinAI</a>,&nbsp;
                    <a href="https://www.redhat.com/en" target="_blank" rel="noopener noreferrer">RedHat</a>, and&nbsp;
                    <a href="https://www.childrenshospital.org/" target="_blank" rel="noopener noreferrer">Boston Children's Hospital</a></p>
```